### ISSUE TITLE
map?

### ISSUE THREAD

**Issue by Znc1996** [csuhan/ReDet#25]
Why is there no map in the test results?

Looking forward for your reply ... thanks

**Comment by csuhan** [comment]
See [GETTING_STARTED.md](https://github.com/csuhan/ReDet/blob/master/GETTING_STARTED.md)

Parse the results.pkl to the format needed for DOTA evaluation
python tools/parse_results.py --config configs/ReDet/ReDet_re50_refpn_1x_dota15.py --type OBB

**Comment by Znc1996** [comment]
detpath = os.path.join(det_dir, r'Task1_{:s}.txt')
    annopath = r'../work_dirs/ReDet_re50_refpn_1x_dota15/Task1_results/Task1_{:s}.txt'
    imagesetfile = r'../data/dota15_1024/test1024/test.txt'

Which label in annopath refers to? 
Does the image refer to the image of the test set?
I haven't been able to test successfully all the time. I really hope you can answer it for me.
Looking forward for your reply ... thanks

**Comment by csuhan** [comment]
You need to use this command 
```bash
python tools/parse_results.py --config configs/ReDet/ReDet_re50_refpn_1x_dota15.py --type OBB
```
to generate formated results, then submit it to [DOTA evaluation server](https://captain-whu.github.io/DOTA/evaluation.html)

**Comment by Znc1996** [comment]
Do I have to test it on this DOTA evaluation server ?
Can I use this code to test the results?（dota_v15_evaluation_task1.py）

**Comment by csuhan** [comment]
By default, we use both `train` and `val` sets for training, and `test` set for testing.
To evaluate it locally, you can train on the `train` set and validate it on the `val` set (with local annotations).
Let me explain the vars below:

```python
# detpath: path to the parsed results.
detpath = os.path.join(det_dir, r'Task1_{:s}.txt')
# annopath: path to the annotation, like: dota15/val/labelTxt/PXXXX.txt
annopath = r'data/dota15/test/dota1_5_labelTxt/{:s}.txt'
# imagesetfile: a txt file recording image names, one image one line, like: 
# P0006
# P0007
# ...
imagesetfile = r'data/dota15/test/testset.txt
```

**Comment by Znc1996** [comment]
I don't quite understand why the test set still has label . What exactly does this label refer to?
annopath = r'data/dota15/test/dota1_5_labelTxt/{:s}.txt'
imagesetfile = r'data/dota15/test/testset.txt
Or does the dota1_5_labelTxt  here refer to the labelTxt in the validation set?

Looking forward for your reply ... thanks.

**Comment by Znc1996** [comment]
I don't quite understand why the test set still has label . What exactly does this label refer to?
annopath = r'data/dota15/test/dota1_5_labelTxt/{:s}.txt'
imagesetfile = r'data/dota15/test/testset.txt
Or does the dota1_5_labelTxt here refer to the labelTxt in the validation set?

Looking forward for your reply ... thanks.
